src/DonationForms/Listeners/StoreCustomFields.php

```diff
@@ -22,10 +23,10 @@ class StoreCustomFields
      * @return void
      * @throws NameCollisionException
      */
-    public function __invoke(DonationForm $form, Donation $donation, array $customFields)
+    public function __invoke(DonationForm $form, Donation $donation, array $customFields, Subscription $subscription = null)
```

Comment:
@jonwaldstein Since we know we're using the subscription value always, let's make it nullable rather than optional:
```php
    public function __invoke(DonationForm $form, Donation $donation, Subscription? $subscription, array $customFields)
```

Reply:
ohhh I didn't realize we could do those now. 7.2 for the win!

Reply:
Resolved here 2438b062